packages/pharos/tokens/assets/icons.json

```diff
@@ -55,6 +55,8 @@
       "menu": { "value": "assets/icons/menu.svg" },
       "move": { "value": "assets/icons/move.svg" },
       "new": { "value": "assets/icons/new.svg" },
+      "pdf": { "value": "assets/icons/pdf.svg" },
+      "ppt": { "value": "assets/icons/ppt.svg" },
```

Comment:
Should these be named something like `filetype-ppt` to (1) have them sort together and (2) reduce any ambiguity about what some might mean?